### PR TITLE
WIP: Update chain.json for IRISnet

### PR DESCRIPTION
**Pull request by zhangyelong**

### CODE REVIEW

File: irisnet/chain.json [hunk 1]
```diff
@@ -1,23 +1,75 @@
...
+      "0.2uiris"
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/irisnet/irishub",
+    "recommended_version": "v1.2.0",
+    "compatible_versions": [
+      "v1.2.0"
+    ],
+    "binaries": {}
```
Comment (sunnya97) [inline irisnet/chain.json:29]:
```suggestion
```

File: irisnet/chain.json [hunk 2]
```diff
@@ -1,23 +1,75 @@
...
+  },
+  "peers": {
+    "seeds": [
+      "6a6de770deaa4b8c061dffd82e9c7f4d40c2165d@seed-1.mainnet.irisnet.org:26656",
+      "a17d7923293203c64ba75723db4d5f28e642f469@seed-2.mainnet.irisnet.org:26656"
```
Comment (sunnya97) [inline irisnet/chain.json:33]: Hi, could you please put these in the format used in the chain.json

Example:
```

            {
                "id": "cfb6f2d686014135d4a6034aa6645abd0020cac6",
                "address": "52.79.88.57:26656",
                "provider": "cosmostation"
            }
```

File: irisnet/chain.json [hunk 3]
```diff
@@ -1,23 +1,75 @@
...
+    "rpc": [
+      "https://rpc-iris.keplr.app"
+    ],
+    "rest": [
+      "https://lcd-iris.keplr.app"
+    ],
+    "grpc": []
+  },
+  "explorers": [
+    "https://irishub.iobscan.io",
```
Comment (sunnya97) [inline irisnet/chain.json:68]: Same with these.  Please use the format in the example (check the Osmosis chain.json for an example)

**Review by sunnya97 — CHANGES_REQUESTED** [review]

**Comment by jackzampolin** [comment]
Closing this as stale, also it looks like the registry has been since updated.

**Pull request closed without merging**